dev/tidb-in-kubernetes/troubleshoot.md

```diff
@@ -350,4 +350,17 @@ tidb:
 
 > **注意：**
 >
-> 以上配置的环境要求：TiDB Operator 1.1 版本或以上。
+> - 以上配置需要 [kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) 配置 `--allowed-unsafe-sysctls=net.*`，如果集群无法配置这个参数，需要在配置 `podSecurityContext` 的同时配置 `annotations`：
+>
+>     ```
+>     tidb:
+>       annotations:
+>         tidb.pingcap.com/sysctl-init: "true"
+>       podSecurityContext:
+>         sysctls:
+>         - name: net.ipv4.tcp_keepalive_time
+>           value: "300"
+>       ...
+>     ```
+>
+> - 以上配置的环境要求：TiDB Operator 1.1 版本或以上。
```

Comment:
because we don't operator 1.1  is not available for the users, hold this after 1.1 is released?

Reply:
I think it's ok to merge it now since we have make it clear for the required version, and we should make the doc ready before operator released.

Reply:
What about taking this content out of the note? It can be too long for a note.